scripts/src/middleware.ts

```diff
@@ -113,9 +119,12 @@ function clientErrorHandler(error: MarketplaceError, req: express.Request, res:
 
 function serverErrorHandler(error: any, req: express.Request, res: express.Response) {
 	const log = req.logger || logger;
-
 	metrics.reportServerError(req.method, req.url);
 
+	const timestamp = Math.floor(Date.now() / 1000);
```

Comment:
`moment.utc().seconds()` or  `new Date().getUTCSeconds();`

Reply:
Both are not good, they give the seconds value of the current time. So you are limited to a timeframe lower than a minute and if more than a minute passes you got a bug on your hands.
If anything `moment().unix()` fits better.